module-bsp/board/rt1051/os/include/FreeRTOSConfig.h

```diff
@@ -78,7 +78,7 @@ extern "C"
 /* Memory allocation related definitions. */
 #define configSUPPORT_STATIC_ALLOCATION  1
 #define configSUPPORT_DYNAMIC_ALLOCATION 1
-#define configTOTAL_HEAP_SIZE            ((size_t)(1024 * 386))
```

Comment:
Are you sure we have space for this?
We have a total of 448K in internal RAM and that memory is shared with audio buffers as far as I know.
`SRAM_DTC (rwx) : ORIGIN = 0x20000000, LENGTH = 0x70000 /* 448K bytes (alias RAM3) */`